dash/include/dash/GlobMem.h

```diff
@@ -105,7 +105,7 @@ class GlobMem
    *       For example, \c dash::LocalAllocator is used in \c dash::Shared
    *       and only called at owner unit.
    */
-  inline GlobMem(
+  explicit GlobMem(
```

Comment:
D'aah, sorry, for whatever reason I mixed things up. I saw the `inline` and was on the constexpr train. Carry on.